nebula-studio/about-studio/st-ug-what-is-graph-studio.md

```diff
@@ -0,0 +1,45 @@
+# What is Nebula Graph Studio
+
+Studio (short for Nebula Graph Studio) is a browser-based visualization tool to manage Nebula Graph databases. It provides graphical user interface for you manipulate graph schemas, import data, explore graph data, and run nGQL statements to query data. With Studio, you can quickly become graph exploration expert from scratch.
+
+## Release distributions
+
+Studio has two release distributions:
```

Comment:
Studio has two releases?

Reply:
> Studio has two releases?

Yes. Docker-based Studio and Studio on Cloud.